src/main-process/parse-command-line.js

```diff
@@ -177,6 +193,22 @@ module.exports = function parseCommandLine (processArgs) {
   }
 }
 
+function readProjectSpecificationSync (filepath, executedFrom) {
+  try {
+    const readPath = path.isAbsolute(filepath) ? filepath : path.join(executedFrom, filepath)
+    const contents = CSON.readFileSync(readPath)
+
+    if (contents.paths == null) {
+      contents.paths = [path.dirname(readPath)]
+    }
+    if (contents.config) {
```

Comment:
So, if `contents` does not have a `config` property, we return undefined? Then we are relying on the fact that `Object.assign({}, undefined)` makes an empty object?

Seems like it would be better to either explicitly return an empty object, or throw.

Reply:
If contents.config doesn't exist, it falls through to the error and does not return undefined.

Reply:
ooh I don't think we should require that to be defined. good catch!